src/traversal/__tests__/diffRelayQuery_connection-test.js

```diff
@@ -9,6 +9,8 @@
  * @emails oncall+relay
  */
 
+/* eslint-disable indent */
```

Comment:
Ok. Got it. I'm worrying about global disabling also. But this rule disabling can't produce real holes for bugs.